cmd/vela-slack/plugin.go

```diff
@@ -189,6 +204,24 @@ func (p *Plugin) Validate() error {
 	return nil
 }
 
+// getRemoteAttachment function to open and parse slack attachment json file into
+// slack webhook message payload.
+func replaceString(bytes []byte, p *Plugin) []byte {
+	// Converts bytes into string and replaces {{ .BuildCreated }}
+	// with a timestamp before returning it back into bytes again.
```

Comment:
```suggestion
// replaceString converts bytes into string and replaces variables
// like {{ .BuildCreated }} with their respective values before
// returning them back into bytes again.
func replaceString(bytes []byte, p *Plugin) []byte {
```

the comment preceding the function didn't seem accurate. the comment within seemed appropriate though. what do you think?

Reply:
oops I forgot to change that, fixed now